internal/pkg/utils/testfs/fs.go

```diff
@@ -1,37 +0,0 @@
-package testfs
-
-import (
-	"fmt"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
-	"github.com/keboola/keboola-as-code/internal/pkg/filesystem/aferofs"
-	"github.com/keboola/keboola-as-code/internal/pkg/log"
-)
-
-func NewBasePathLocalFs(basePath string) filesystem.Fs {
-	fs, err := aferofs.NewLocalFs(log.NewNopLogger(), basePath, `/`)
-	if err != nil {
-		panic(err)
-	}
-	return fs
-}
-
-func NewMemoryFs() filesystem.Fs {
```

Comment:
Odstranil som balicek `testFs`.
Sluzil na vytvaranie FS v testoch, ked napr nepotrebujes definovat logger alebo kontrolovat chybu.
Problem bol, ze niekde sa to v testoch pouzivalo a niekde nie.

Nahradil som to vylepsenim balika `aferofs` aby si nemusel vzdy zadavat `logger` a `workingDir` ak to nepotrebujes.